### PR TITLE
New cheat sheet: Wunderlist Mac

### PR DESCRIPTION
**Pull request by mayank7**
New cheat sheet: Wunderlist Mac

IA Page: https://duck.co/ia/view/wunderlist_mac_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mayank7** [comment]
@moollaza @MrChrisW This is a new cheat sheet, you can put the label New Instant Answer

File: share/goodie/cheat_sheets/json/wunderlist-mac.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{
+    "id": "wunderlist_mac",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wunderlist-mac.json]: Please change to `wunderlist_mac_cheat_sheet`. **_All_** Cheat Sheets must have `_cheat_sheet` in the ID

**Comment by moollaza** [comment]
@mayank7 the content looks good. Once the `id` is fixed in the JSON this is good for merge :+1:

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=wunderlist mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wunderlist_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=wunderlist mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wunderlist_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=wunderlist mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wunderlist_mac_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=wunderlist mac cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/wunderlist_mac_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@mayank7 Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@mayank7 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**